target/linux/ipq40xx/files-6.1/arch/arm/boot/dts/qcom-ipq4019-whw03.dts

```diff
@@ -0,0 +1,416 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Linksys WHW03 (Velop)";
+	compatible = "linksys,whw03", "qcom,ipq4019";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_red;
+		led-running = &led_green;
+		led-upgrade = &led_red;
+	};
+
+	// Default bootargs includes rootfstype=ext4 and needs to be overriden.
+	chosen {
+		bootargs-append = " rootfstype=squashfs clk_ignore_unused";
+		stdout-path = &blsp1_uart1;
```

Comment:
Why not `stdout-path = "serial0:115200n8";` ?

Reply:
It works most likely because bootloader is passing `console=ttyMSM0` via cmdline.
I would prefer to have the serial0 alias set in aliases and then chosen used to indicate it.

So add:
`serial0 = &blsp1_uart1;`

To the aliases node, and for stdout path use:
`stdout-path = "serial0:115200n8";`

Reply:
ok got it, thanks. done!

Reply:
NOTE: rolled back the changes and reopened (see below)

Reply:
> Hm, suddenly MSM serial fails probing for whatever reason and thus you will get the crash.
You can basically drop stdout-path and serial0 alias completely as bootloader is passing the desired console to spare us all from actually debugging where it fails

done, removed the items and tested an image: console works fine.